forge-gui/res/cardsfolder/upcoming/the_eternal_wanderer.txt

```diff
@@ -0,0 +1,17 @@
+Name:The Eternal Wanderer
+ManaCost:4 W W
+Types:Legendary Planeswalker
+Loyalty:5
+S:Mode$ AttackRestrict | MaxAttackers$ 1 | DefenderDefined$ Card.Self | Description$ No more than one creature can attack CARDNAME each combat.
+A:AB$ ChangeZone | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | Origin$ Battlefield | Destination$ Exile | ValidTgts$ Artifact,Creature | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target artifact or creature | RememberChanged$ True | SubAbility$ DBDelTrig | SpellDescription$ Exile up to one target artifact or creature. Return that card to the battlefield under its owner's control at the beginning of that player's next end step.
+SVar:DBDelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | DelayedTriggerDefinedPlayer$ TargetedOwner | Execute$ TrigReturn | RememberObjects$ RememberedLKI | SubAbility$ DBCleanup | TriggerDescription$ Return that card to the battlefield under its owner's control at the beginning of that player's next end step. | StackDescription$ Return that card to the battlefield under its owner's control at the beginning of that player's next end step.
```

Comment:
This should be done with ValidPlayer

I'll add PlayerProperty IsTriggerRemembered in my next PR

Reply:
✅